### PR TITLE
Rephrase "TL;DR"

### PR DESCRIPTION
**Pull request by LukasReschke**
"TL;DR" seems not to be understood or respected by everyone. Maybe we should do a try with "Important". Open for alternative suggestions.

Any other ideas?

cc @DeepDiver1975 @jospoortvliet @jancborchardt @karlitschek

### CODE REVIEW

**Comment by karlitschek** [comment]
yes. definitely

**Comment by jancborchardt** [comment]
Maybe just call it »Short version«, that’s even simpler. And good call to simplify the wording!

**Comment by LukasReschke** [comment]
Changed to short version as suggested by @jancborchardt as well as added a little sentence stating that the issue template should be used all the time.

**Comment by nickvergessen** [comment]
no test coverage hopefully :+1:
